internal/pacseek/display.go

```diff
@@ -241,13 +241,14 @@ func (ps *UI) displayHelp() {
 		SetCellSimple(2, 0, "Up/Down: Navigate within package list").
 		SetCellSimple(3, 0, "Shift+Left/Right: Change size of package list").
 		SetCellSimple(4, 0, "CTRL+S: Open/Close settings").
-		SetCellSimple(5, 0, "CTRL+N: Show these instructions").
-		SetCellSimple(6, 0, "CTRL+U: Perform sysupgrade").
-		SetCellSimple(7, 0, "CTRL+A: Perform AUR upgrade (if configured)").
-		SetCellSimple(8, 0, "CTRL+W: Wipe cache").
-		SetCellSimple(9, 0, "CTRL+P: Show PKGBUILD for selected package").
-		SetCellSimple(10, 0, "CTRL+G: Show list of upgradeable packages").
-		SetCellSimple(11, 0, "CTRL+L: Show list of all installed packages").
+		SetCellSimple(5, 0, "CTRL+O: Opens selected package repo url").
```

Comment:
I would put that at position 10, next to CTRL+P and rephrase it to `CTRL+O: Show URL for selected package` to be in line with CTRL+P...